sdk/identity/azure-identity/azure/identity/_credentials/linux_vscode_adapter.py

```diff
@@ -0,0 +1,65 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import os
+import json
+import ctypes as ct
+from .._constants import VSCODE_CREDENTIALS_SECTION
+
+
+def _c_str(string):
+    return ct.c_char_p(string.encode('utf-8'))
```

Comment:
It seems this is used for service name and account name - service name is probably okay, but is there any change that an account name might not be utf8?

Reply:
This is a good question and I don't have an answer. I opened #11135 to track it.

Reply:
I assume that would be utf8

Reply:
Yes. I discussed with Anna offline. We assume this is utf8. We will try if we can find a way to test it.